spec/features/idv/inherited_proofing/analytics_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+feature 'Inherited Proofing Analytics Regression', js: true do
+  include InheritedProofingHelper
+
+  let(:user) { user_with_2fa }
+  let(:fake_analytics) { FakeAnalytics.new }
+  let(:auth_code) { Idv::InheritedProofing::Va::Mocks::Service::VALID_AUTH_CODE }
+  # rubocop:disable Layout/LineLength
+  let(:happy_path_events) do
+    {
+      'Idv: inherited proofing get started visited' => { flow_path: 'standard', step: 'get_started', step_count: 1, analytics_id: 'Inherited Proofing' },
+      'Idv: inherited proofing get started submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'get_started', step_count: 1, analytics_id: 'Inherited Proofing' },
+      'Idv: inherited proofing agreement visited' => { flow_path: 'standard', step: 'agreement', step_count: 1, analytics_id: 'Inherited Proofing' },
+      'Idv: inherited proofing agreement submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'agreement', step_count: 1, analytics_id: 'Inherited Proofing' },
+      'Idv: inherited proofing verify wait step visited' => { flow_path: 'standard', step: 'verify_wait', step_count: 1, analytics_id: 'Inherited Proofing' },
+      'Idv: inherited proofing verify wait submitted' => { success: true, errors: {}, step: 'verify_wait_step_show', analytics_id: 'Inherited Proofing' },
+      'Idv: inherited proofing verify visited' => { flow_path: 'standard', step: 'verify_info', step_count: 1, analytics_id: 'Inherited Proofing' },
+      'Idv: inherited proofing verify submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'verify_info', step_count: 1, analytics_id: 'Inherited Proofing' },
+    }
+  end
+  # let(:gpo_path_events) do
+  #   {
+  #     'IdV: intro visited' => {},
+  #     'IdV: doc auth welcome visited' => { flow_path: 'standard', step: 'welcome', step_count: 1 },
+  #     'IdV: doc auth welcome submitted' => { success: true, errors: {}, flow_path: 'standard', step: 'welcome', step_count: 1 },
+  #     'IdV: doc auth agreement visited' => { flow_path: 'standard', step: 'agreement', step_count: 1 },
```

Comment:
I'd make sure we remove these commented out blocks before merging

Reply:
Good catch, I meant to delete that. Thank you

Reply:
Removed. Thanks again for catching that.